utils/firestore.js

```diff
@@ -111,20 +113,6 @@ const updateResponse = async (data, uid) => {
     }
 }
 
-const incrementCounter = async (field, siteCode) => {
```

Comment:
assuming these weren't being used anywhere in the code base?

Reply:
It's not, so I removed it.